.changeset/six-hairs-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Now 'no shipping method' warning in draft order creation has an improved, clearer message
```

Comment:
_Now you can see the reason why shipping methods are incorrect when creating draft orders. This means creating new draft orders will pop up an error with the details of incorrect shipping._